lib/segment/analytics.ex

```diff
@@ -164,6 +164,6 @@ defmodule Segment.Analytics do
 
   @spec call(Segment.segment_event()) :: :ok
   def call(event) do
-    Segment.Config.service().call(event)
+    if Segment.Config.enabled?, do: Segment.Config.service().call(event)
```

Comment:
else return `:ok` so to follow the spec?

```suggestion
  case Segment.Config.enabled? do
    true -> Segment.Config.service().call(event)
    _ -> :ok
  end
```

Reply:
I prefer a simple else.